examples/conftest.yml

```diff
@@ -0,0 +1,91 @@
+# This example shows how to use compost with **Atlantis v0.18**, Conftest, and Infracost to enforce cost policies.
+
+# To run this:
+# 1. Add the yaml contents of this file to your `repos.yaml` or `atlantis.yaml` server side config file.
+# 2. On the Atlantis server, export env vars for:
+#    export GITHUB_TOKEN=<your-github-token>
+#    export INFRACOST_API_KEY=<your-infracost-api-token>
+# 3. Run the `infracost/infracost-atlantis` image, which includes the Infracost and Compost CLIs in addition to Atlantis:
+#    docker run -p 4141:4141 -e GITHUB_TOKEN=$GITHUB_TOKEN -e INFRACOST_API_KEY=$INFRACOST_API_KEY \
```

Comment:
We only need `-e GITHUB_TOKEN=$GITHUB_TOKEN` if we're posting a comment to GITHUB

Reply:
Ah ok, I thought Atlantis was using that to post its comments.

Reply:
Atlantis uses the value passed in `--gh-token=$GITHUB_TOKEN`, which they unfortunately don't expose to us so we need this env variable set explicitly for our comments.